### PR TITLE
don't set executable when not using a shell

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

File: lib/vsc/utils/fancylogger.py [hunk 1]
```diff
@@ -740,9 +740,7 @@ def setLogLevel(level):
     """
     if isinstance(level, basestring):
         level = getLevelInt(level)
-
-    logger = getLogger(fname=False, clsname=False)
-
+    logger = logging.getLogger()
```
Comment (boegel) [inline lib/vsc/utils/fancylogger.py]: @JensTimmerman What's the point of this, in this PR? It even makes tests fail it seems...

File: setup.py [hunk 2]
```diff
@@ -44,7 +44,7 @@
 ]
 
 PACKAGE = {
-    'version': '2.7.3',
+    'version': '2.7.5',
```
Comment (boegel) [inline setup.py]: why not `.4`?

File: test/run.py [hunk 3]
```diff
@@ -92,6 +92,15 @@ def test_noshell_glob(self):
         self.assertEqual(ec, 0)
         self.assertTrue(all(x in output.lower() for x in ['__init__.py', 'testmodule.py', 'testmodulebis.py']))
 
+    def test_noshell_executable(self):
+        ec, output = run("""%s -c 'print ("foo")'""" % sys.executable)
+        self.assertEqual(ec, 0)
+        self.assertTrue('foo' in output)
+        ec, output = run_simple([sys.executable, '-c', 'print ("foo")'])
```
Comment (boegel) [inline test/run.py]: `run_simple` expects a string, while `run` expects and array (you have them the other way around)?

**Comment by boegel** [comment]
better/proper fix in #267...

File: test/run.py (hunk 3 above)
Comment (JensTimmerman) [reply test/run.py]: no, run_simple also works with an array

Comment (JensTimmerman) [reply test/run.py]: and run also works with a string, but it's going to be very very escaped

**Pull request closed without merging**